From 0b19078b2bb588e5664a199025de53db6c7aa8c1 Mon Sep 17 00:00:00 2001
From: Daniel van Vugt <daniel.van.vugt@canonical.com>
Date: Fri, 6 Sep 2019 17:42:19 +0800
Subject: [PATCH 16/30] clutter/stage-cogl: Use portable formatting for 64-bit
 integers

On 64-bit systems int64_t is %ld but on 32-bit systems it is %lld.

https://gitlab.gnome.org/GNOME/mutter/merge_requests/778
---
 clutter/clutter/cogl/clutter-stage-cogl.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/clutter/clutter/cogl/clutter-stage-cogl.c b/clutter/clutter/cogl/clutter-stage-cogl.c
index 7dfd9b839..4810e42a4 100644
--- a/clutter/clutter/cogl/clutter-stage-cogl.c
+++ b/clutter/clutter/cogl/clutter-stage-cogl.c
@@ -204,7 +204,7 @@ clutter_stage_cogl_schedule_update (ClutterStageWindow *stage_window,
   if (max_render_time_allowed <= 0)
     {
       g_warning ("Unsupported monitor refresh rate detected. "
-                 "(Refresh rate: %.3f, refresh interval: %ld)",
+                 "(Refresh rate: %.3f, refresh interval: %" G_GINT64_FORMAT ")",
                  refresh_rate,
                  refresh_interval);
       stage_cogl->update_time = now;
-- 
2.23.0

